dds_cli/__main__.py

```diff
@@ -79,7 +82,8 @@
     "\n[green](  ) ) (  (  )[/]   [bold]SciLifeLab Data Delivery System",
     "\n[green] ︶  (  ) ) ([/]    [blue][link={0}]{0}[/link]".format(dds_cli.__url__),
     f"\n[green]      ︶ (  )[/]    [dim]Version {dds_cli.__version__}",
-    "\n[green]          ︶\n",
+    "\n[green]          ︶",
+    f"\n[green]Logged in:[/] [red]{username}",
```

Comment:
Maybe we could change this to something like 
* User: 
* Authenticated user:
* Running command as: 
* Using dds-cli as
Not sure but "Logged in" doesn't feel right to me